src/utils/math-utils.ts

```diff
@@ -34,3 +34,19 @@ export const calculateBoundedPosition = (left: number, top: number, width: numbe
 export const getRandomInteger = (min: number, max: number) => {
   return Math.floor(Math.random() * (max - min + 1) ) + min;
 };
+
+// c.f. https://github.com/Daplie/knuth-shuffle
```

Comment:
A shorter alternative from [this StackOverflow post](https://stackoverflow.com/a/12646864):
```typescript
function shuffleArray(array) {
    for (let i = array.length - 1; i > 0; i--) {
        const j = Math.floor(Math.random() * (i + 1));
        [array[i], array[j]] = [array[j], array[i]];
    }
}
```

Reply:
I like it! Updated.